includes/Modules/Search_Console.php

```diff
@@ -362,50 +362,35 @@ protected function create_search_analytics_data_request( array $args = array() )
 	/**
 	 * Checks whether Search Console data exists for the given post.
 	 *
-	 * The result of this query is stored in a transient which is refreshed every 2 hours.
+	 * The result of this query is stored in a transient.
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param int $post_id Post ID.
+	 * @param string $current_url The current url.
 	 * @return bool True if Search Console data exists, false otherwise.
 	 */
-	protected function has_data_for_post( $post_id ) {
-		if ( ! $post_id ) {
+	protected function has_data_for_url( $current_url ) {
+		if ( ! $current_url ) {
 			return false;
 		}
 
-		$transient_key = 'googlesitekit_sc_has_data_for_post_' . $post_id;
+		$transient_key = md5( 'gogolesitekit_sc_data_' . $current_url );
```

Comment:
Two minor issues here:
* A typo in the string.
* Let's only `md5` the `$current_url` and keep the `googlesitekit_sc_data_` prefix in clear text.

Reply:
👍  will adjust

Reply:
Adjusted and corrected typo in https://github.com/google/site-kit-wp/pull/543/commits/fce143f6c305633b2250727e9a6cb1f9db3f691e